Documentation/Books/Manual/Administration/ManagingUsers/README.md

```diff
@@ -139,21 +139,25 @@ from the database access level in the `_system` database, it is
 *Administrate*. Note that this means that database access level
 *Access* does not grant a user server access level *Administrate*.
 
+When a user creates a database the access level of the user for that database is set to *Administrate*. The same is true for creating a collection, in this case the user get *Read/Write* access to the collection.
+
```

Comment:
Please also write down the default access level to collections.
Namely if a user creates a database he does have default `no access` to all collections

Reply:
This is wrong. He does have *read/write* access to all collections within a database where he has *administrate* rights